src/python/pants/notes/2.9.x.md

```diff
@@ -1,5 +1,37 @@
 # 2.9.x Release Series
 
+## 2.9.0.dev1 (Nov 13, 2021)
+
+### New Features
+
+* Add `instructions` field to `docker_image` to support generating the Dockerfile ([#13386](https://github.com/pantsbuild/pants/pull/13386))
+
+* Add `--unowned-dependency-behavior` option for Python ([#13491](https://github.com/pantsbuild/pants/pull/13491))
+
+### User API Changes
+
+* Upgrade default pylint to v2.11.1 ([#13585](https://github.com/pantsbuild/pants/pull/13585))
+
+### Plugin API Changes
+
+* [plugin-api] Add `default_glob_match_error_behavior` to `SourcesField`. ([#13578](https://github.com/pantsbuild/pants/pull/13578))
+
+### Bug fixes
+
+* Fix `--no-print-stacktrace`. ([#13539](https://github.com/pantsbuild/pants/pull/13539))
+
+### Performance
+
+* Do not invalidate parent paths when only file content has changed ([#13566](https://github.com/pantsbuild/pants/pull/13566))
+
+* Fix caching of binary lookups and avoid re-running `pytest` setup for environment changes ([#13559](https://github.com/pantsbuild/pants/pull/13559))
+
+* Fix slow performance from recursive `__repr__` and `__hash__` for Go ([#13492](https://github.com/pantsbuild/pants/pull/13492))
+
```

Comment:
Does `Go: improve performance of first-party package analysis (#13476)` belong here? I thought we now included Go in release notes, just not JVM/

Reply:
Yeah, we do now include Go in release notes. I put it in performance for the 2.8 cherry-pick